modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/grafanaagent.yaml

```diff
@@ -22,3 +22,9 @@ spec:
     instanceSelector:
       matchLabels:
         agent: grafana-agent-logs
+  resources:
+    requests:
```

Comment:
I guess we skip limits due to that we can't configure the log agents and the metrics agents separately. Have you created any issue for this upstream? If so please link it and create a new issue in the terraform modules repo so we can keep a track of it.

Reply:
So far I have not seen any need for separate limits since VPA has recommended the same for both in the same environment. Let's do that when we see a need for it

Reply:
Since things can take a long time upstream I think it would be good to create the issue and I don't think this will be true in the long run. Especially in bigger clusters with lots and lots of metrics.

We can at least see what the upstream community thinks about it.

Reply:
https://github.com/grafana/agent/issues/1348